lib/util.js

```diff
@@ -871,12 +878,13 @@ exports.getProxyFromEnv = function (isHttps = true) {
   Logger.getInstance().debug(`Util.getProxyEnv: Using ${protocol.toUpperCase()}_PROXY from the environment variable`);
   if (proxyFromEnv.indexOf('://') === -1) {
     Logger.getInstance().info('Util.getProxyEnv: the protocol was missing from the environment proxy. Use the HTTP protocol.');
```

Comment:
We should change this to log proper protocol

Reply:
this line is not part of this change; but the earlier implementation with #938 . 
It was agreed, that we follow the practice set in the industry and the other drivers, and will use the HTTP protocol if the user does not set it in the envvar.